mappings/net/minecraft/entity/ai/brain/MemoryModuleType.mapping

```diff
@@ -5,6 +5,9 @@ CLASS net/minecraft/unmapped/C_vbbyoqyw net/minecraft/entity/ai/brain/MemoryModu
 	COMMENT
 	COMMENT @see Brain#memories
 	COMMENT @see Memory
+	FIELD f_baqvtxeq UNREACHABLE_TONGUE_TARGETS Lnet/minecraft/unmapped/C_vbbyoqyw;
```

Comment:
it's funny to see this without context :p

Reply:
it is, i dont think theres a better name though, funny is good sometimes